### PR TITLE
Ketchup as fake blood!

### PR DESCRIPTION
**Pull request by ZeWaka**
When you get more than 5u of ketchup on the ground, you get 'blood' that's actually ketchup.
It's slightly brighter and has an obvious examinetext.

http://forum.ss13.co/showthread.php?tid=5949

### CODE REVIEW

File: code/datums/chemistry/Reagents-FoodDrink.dm [hunk 1]
```diff
@@ -1821,6 +1821,12 @@ datum
 			fluid_g = 0
 			fluid_b = 0
 			transparency = 255
+			reaction_turf(var/turf/T, var/volume) //Makes the kechup splats
```
Comment (Iamgoofball) [inline code/datums/chemistry/Reagents-FoodDrink.dm:1824]: ew, relative pathing

File: code/datums/chemistry/Reagents-FoodDrink.dm [hunk 2]
```diff
@@ -1821,6 +1821,12 @@ datum
 			fluid_g = 0
 			fluid_b = 0
 			transparency = 255
+			reaction_turf(var/turf/T, var/volume) //Makes the kechup splats
+				src = null
```
Comment (Iamgoofball) [inline code/datums/chemistry/Reagents-FoodDrink.dm:1825]: do this after the rest, could cause problems

**Comment by ninjacha** [comment]
the entire code is relative pathing.

**Comment by volundr-** [comment]
See [Readme](https://github.com/goonstation/goonstation-2016/blob/master/README.md), thanks!

**Pull request closed without merging**